lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
+
+  extensions:
+    - .coffee
+    - .js
+    - .jsx
+    - .ts
+    - .vue
+    - .sass
+    - .scss
+    - .css
+    - .png
+    - .svg
+    - .gif
+    - .jpeg
+    - .jpg
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+  paths:
+    output: packs-test
+    manifest: manifest-test.json
```

Comment:
Don't think we need a different `manifest` filename if the files are all in separate dir.